inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -447,9 +447,11 @@ private void actionAccept(AjaxRequestTarget aTarget) throws AnnotationException,
         int end = acceptedRecommendation.getOffset().getEndCharacter();
         int id = adapter.add(annotatorState, jCas, begin, end);
         for (AnnotationFeature feature : annotationService
-                .listAnnotationFeature(selectedLayer.getObject())) {
-            adapter.setFeatureValue(annotatorState, jCas, id, feature,
+            .listAnnotationFeature(selectedLayer.getObject())) {
```

Comment:
Should be using `annotationService.getFeature(acceptedRecommendation.getFeature(), selectedLayer)` instead of looping over all features.